src/containers/ProfilePage/ProfilePage.module.css

```diff
@@ -203,3 +203,105 @@
 .desktopReviewsTabNav {
   margin-bottom: 32px;
 }
+
+.sectionText {
+  padding: 0 24px;
+  margin-bottom: 30px;
+
+  @media (--viewportMedium) {
+    padding: 0;
+    margin-bottom: 40px;
+  }
+}
+
+/* 
+ * Custom field component classes 
+ */
+
+.sectionHeading,
+.sectionHeadingWithExtraMargin {
+  /* Font for sections titles */
+  composes: p from global;
+  padding: 0;
+  font-weight: var(--fontWeightBold);
+
+  margin-top: 0;
+  margin-bottom: 12px;
+
+  @media (--viewportMedium) {
+    margin-top: 0;
+    margin-bottom: 8px;
+  }
+}
+
+.sectionHeadingWithExtraMargin {
+  @media (--viewportMedium) {
+    margin-bottom: 16px;
+  }
+}
+
+.text {
+  font-weight: var(--fontWeightRegular);
+  padding: 1px 0 5px 0;
+  margin: 0;
+
+  /* Preserve newlines, but collapse other whitespace */
+  white-space: pre-line;
+
+  @media (--viewportMedium) {
+    padding: 3px 0 5px 0;
+    margin: 0;
+  }
+}
+
+.ingress {
```

Comment:
I don't this is used in these components

Reply:
SectionTextMaybe has the option of showing the text block as ingress with this class, so if we want to remove that capability from the ProfilePage version, then we can also remove this class.